packages/imguiml/imguiml.v1.90.6/opam

```diff
@@ -0,0 +1,68 @@
+opam-version: "2.0"
+synopsis: "ImGUI bindings to OCaml"
+description: """
+Imguiml provides bindings to ImGui and ImGui_Impl for GLFW and OpenGL3 in OCaml,
+through cimgui.
+The version follows the version of ImGui for which the package is providing
+bindings.
+The docking branch is the only one for which we provide bindings.
+"""
+maintainer: ["Emilien Lemaire <emilien.lem@icloud.com>"]
+authors: ["Emilien Lemaire <emilien.lem@icloud.com>"]
+license: "MIT"
+tags: ["graphics" "imgui" "ctypes" "bindings"]
+homepage: "https://github.com/emilienlemaire/imguiml"
+doc: "https://emilienlemaire.github.io/imguiml"
+bug-reports: "https://github.com/emilienlemaire/imguiml/issues"
+depends: [
+  "ocaml" {>= "4.08"}
+  "dune" {>= "3.15"}
+  "goblint-cil" {>= "2.0.0"}
+  "ctypes"
+  "ctypes-foreign" {>= "0.21.1"}
+  "odoc" {with-doc}
+  "sherlodoc" {with-doc}
+  "ppx_expect" {with-test}
+  "conf-glew"
+  "conf-glfw3"
+]
+depexts: [
+  ["gcc" "cmake"]
+]
```

Comment:
```suggestion
]
```
A c compiler is already installed, the depexts as they are are not really used.
If you want to enforce `gcc` use `"conf-gcc"` as a dependency and if you want to enforce the presence of `clang` use `"conf-clang"` as a dependency